### PR TITLE
use MESOS_HTTP{,s} healthchecks everywhere

### PR DESCRIPTION
**Pull request by Rob-Johnson**

### CODE REVIEW

**Comment by solarkennedy** [comment]
Standard pin stuff, and the yelp-main healthcheck needs to be adjusted.

**Comment by Rob-Johnson** [comment]
done @mattmb

File: paasta_tools/utils.py [hunk 1]
```diff
@@ -1630,6 +1631,13 @@ def get_deployd_log_level(self) -> str:
         :return: string name of python logging level, e.g. INFO, DEBUG etc.
         """
         return self.config_dict.get("deployd_log_level", 'INFO')
+    
+    def get_use_mesos_healthchecks(self) -> bool:
+        """Get the log level for paasta-deployd
+
+        :return: string name of python logging level, e.g. INFO, DEBUG etc.
+        """
+        return self.config_dict.get("deployd_log_level", 'INFO')
```
Comment (mattmb) [inline paasta_tools/utils.py]: woops?

Comment (Rob-Johnson) [reply paasta_tools/utils.py]: ha, this is why we code review

Comment (Rob-Johnson) [reply paasta_tools/utils.py]: this is why we code review

**Pull request merged**